Tiltfile

```diff
@@ -0,0 +1,25 @@
+# Eraser Tiltfile.
+## https://docs.tilt.dev/install.html
+### This is a python file. Change Text editor file association for Syntax highlighting.
+
+print("""
+-----------------------------------------------------------------
+ERASER(📒 ✏️) Assumes docker installed, a kind cluster instantiated and kubectl in the right context. More on Tilt: https://docs.tilt.dev/install.html
+-----------------------------------------------------------------
+""".strip())
+
+docker_build('eraser-build', context='.')
```

Comment:
I am a little confused, what is `eraser-build` here? how do we build a new image and push to kind registry here?

example: https://github.com/open-policy-agent/gatekeeper/pull/1895/

Reply:
wouldn't we just deploy 0.1.0 every time since pull policy is always?

Reply:
You are correct, I addressed in latest update.